composer.json

```diff
@@ -54,7 +60,8 @@
         "jms/serializer-bundle":          "Add support for advanced serialization capabilities, recommended, requires ^1.0",
         "symfony/expression-language":    "Add support for using the expression language in the routing, requires ^2.7|^3.0",
         "symfony/serializer":             "Add support for basic serialization capabilities and xml decoding, requires ^2.7|^3.0",
-        "symfony/validator":              "Add support for validation capabilities in the ParamFetcher, requires ^2.7|^3.0"
+        "symfony/validator":              "Add support for validation capabilities in the ParamFetcher, requires ^2.7|^3.0",
+        "sllh/php-cs-fixer-styleci-bridge": "To auto configure PHP-CS-Fixer from StyleCI config file, requires ^1.3"
```

Comment:
can be moved back to ``require-dev`` once https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/pull/49 is merged